From 635ef4f17d921b7e9ccb693582a3122a22fdc099 Mon Sep 17 00:00:00 2001
From: Yanjiang Jin <yanjiang.jin@windriver.com>
Date: Tue, 8 Aug 2017 16:59:55 +0800
Subject: [PATCH 182/184] octeon-error-tree.c: update the array name
 octeon_78xx_error_array

The array name error_array_cn78xxp1 is changed in SDK 3.1.2-568, follow it.

Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/cavium-octeon/octeon-error-tree.c | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/arch/mips/cavium-octeon/octeon-error-tree.c b/arch/mips/cavium-octeon/octeon-error-tree.c
index 823d839..0ecf8b3 100644
--- a/arch/mips/cavium-octeon/octeon-error-tree.c
+++ b/arch/mips/cavium-octeon/octeon-error-tree.c
@@ -424,9 +424,15 @@ static int __init octeon_error_tree_init78(void)
 
 	octeon_78xx_error_array = array->array;
 #if defined(CONFIG_KEXEC) || defined(CONFIG_CRASH_DUMP)
-	for (i = 0; error_array_cn78xxp1[i].intsn < 0xfffff; i++)
-		if (error_array_cn78xxp1[i].error_group != CVMX_ERROR_GROUP_ETHERNET)
-			octeon_ciu3_errbits_disable_intsn(0, error_array_cn78xxp1[i].intsn);
+	for_each_online_node(node)
+	for (i = 0; octeon_78xx_error_array[i].intsn < 0xfffff; i++) {
+		enum cvmx_error_groups group;
+		group = octeon_78xx_error_array[i].error_group;
+		if (group != CVMX_ERROR_GROUP_ETHERNET
+		    && group != CVMX_ERROR_GROUP_ILK
+		    && group != CVMX_ERROR_GROUP_ILA)
+			octeon_ciu3_errbits_disable_intsn(node, octeon_78xx_error_array[i].intsn);
+	}
 #endif
 	for_each_online_node(node)
 		for (i = 0; octeon_78xx_error_array[i].intsn < 0xfffff; i++) {
-- 
1.9.1

